entry.py

```diff
@@ -50,53 +51,39 @@
     sys.exit(1)
 
 
-@logged
-def main():
-    start_time: datetime = datetime.now()
-    logging.info(f'Starting new run at {start_time}')
+def main() -> None:
+    start_time: datetime = datetime.now(tz=utc)
+    LOGGER.info(f'Starting new run at {start_time}')
 
     reports: List[Report] = list(Report.objects.all())
-    logging.info(reports)
+    LOGGER.debug(reports)
 
     exams: List[Exam] = list(Exam.objects.all())
-    logging.info(exams)
-
-    for exam in exams:
-        LOGGER.info(f'Processing Exam: {exam.name}')
-        exam_orca: ScoresOrchestration = ScoresOrchestration(API_UTIL, exam)
-        exam_orca.main()
-
-    # spe_report: SPESummaryReport = SPESummaryReport()
-    # next_query_date: Dict[str, str] = score_handler.orchestrator()
-    # if next_query_date:
-    #     try:
-    #         # sorting the dict so that it always be in format {'place':2019-09-12T14:04:42Z, 'val':2019-09-15T14:04:42Z}
-    #         sorted_next_query_date = dict(sorted(next_query_date.items()))
-    #         query_date_holder.store_next_query_date(sorted_next_query_date)
-    #         spe_report.next_stored_persisted_date = sorted_next_query_date
-    #     except (OSError, IOError, Exception) as e:
-    #         logging.error(f"""error storing the latest assignment submitted date due to {e} 
-    #                     stored date in persisted storage is {stored_submission_date}""")
-    #         spe_report.next_stored_persisted_date = stored_submission_date
-    # else:
-    #     logging.info(f"No next Query date to update either no new scores or error in sending first score")
-    #     spe_report.next_stored_persisted_date = stored_submission_date
-
-    end_time: datetime = datetime.now()
-    delta: datetime = end_time - start_time
-
-    # spe_report.start_time = start_time
-    # spe_report.end_time = end_time
-    # spe_report.elapsed_time = delta
-
-    # if spe_report.is_any_scores_received():
-    #     logging.info('Sending Email....')
-    #     spe_report.send_email()
-    # else:
-    #     logging.info('Not Sending Email since no scores received')
-
-    logging.info(f'The run ended at {end_time}')
-    logging.info(f'Duration of run: {delta}')
+    LOGGER.debug(exams)
+
+    for report in reports:
+        reporter: Reporter = Reporter(report)
+        for exam in report.exams.all():
+            LOGGER.info(f'Processing Exam: {exam.name}')
+            exam_start_time = datetime.now(tz=utc)
+            exam_orca: ScoresOrchestration = ScoresOrchestration(API_UTIL, exam)
+            exam_orca.main()
+            exam_end_time = datetime.now(tz=utc)
+            metadata: Dict[str, datetime] = {
+                'start_time': exam_start_time,
+                'end_time': exam_end_time,
+                'datetime_filter': exam_orca.sub_time_filter
```

Comment:
Nitpick: I feel name of `datatime_filter` can be like `sub_datatime_filter` as the dict variable referenced in reporter will make more sense

Reply:
Sure, I can do that.

Reply:
I renamed to `sub_time_filter`, as that means it will be the same as the property on `Orchestration` that is referenced in the assignment. See https://github.com/tl-its-umich-edu/placement-exams/pull/14/commits/494fa483cd87ad157ecf810431f4e87d8879f5aa